### PR TITLE
Disable the ParallelizableTestsAnalyer for temporary

### PR DESCRIPTION
**Pull request by LiyueWang**

### CODE REVIEW

**Comment by LiyueWang** [comment]
Do I need to bump the version?

File: src/D2L.CodeStyle.TestAnalyzers/ParallelizableTests/ParallelizableTestsAnalyzer.cs [hunk 1]
```diff
@@ -8,8 +8,9 @@
 using Microsoft.CodeAnalysis.Diagnostics;
 
 namespace D2L.CodeStyle.TestAnalyzers.ParallelizableTests {
-    [DiagnosticAnalyzer( LanguageNames.CSharp )]
-    public sealed class ParallelizableTestsAnalyzer : DiagnosticAnalyzer {
+	//Disable the ParallelizableTestsAnalyzer for temporary, too many tests need to be fixed for this property, defer to do it after upgrading to NUnit3
+	//[DiagnosticAnalyzer( LanguageNames.CSharp )]
+	public sealed class ParallelizableTestsAnalyzer : DiagnosticAnalyzer {
```
Comment (saquibmian) [inline src/D2L.CodeStyle.TestAnalyzers/ParallelizableTests/ParallelizableTestsAnalyzer.cs]: nit: formatting

**Comment by saquibmian** [comment]
@LiyueWang yes, you'll need to bump the version

**Comment by saquibmian** [comment]
LGTM

**Pull request merged**